src/controllers/graphql/resolvers/post/postResolver.js

```diff
@@ -1,28 +1,110 @@
-const { AuthenticationError } = require('apollo-server');
 const db = require('../../connectors');
-
 const Post = db.sequelize.models.post;
-const ReplyService = require('../../../../services/replyService');
 const PostService = require('../../../../services/postService');
 const { FETCH_TOP_X_POSTS } = require('../../../../utils/constants');
+const {
+  AuthenticationError,
+  gql,
+  makeExecutableSchema,
+} = require('apollo-server');
+
+const typeDefs = gql`
+  scalar Date
+
+  type Query {
+    getAllPosts: [Post]
+    getPostById(id: ID!): Post
+    getFollowerPosts: [Post]
+    getHundredxPosts: [Post]
+    # getTextPosts: [Post]
```

Comment:
I commented out the queries we haven't implemented yet so it's easier to differentiate between what's done and what needs to be done

Reply:
cool beans